mobile/src/components/common/table/table.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactElement } from 'react';
+import {
+  Cell,
+  Rows,
+  Table as UITable,
+  TableWrapper,
+} from 'react-native-table-component';
+
+import { ScrollView, View } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props<Data extends Record<string, unknown>> = {
+  header: Data[];
+  data: Data[][];
```

Comment:
Create common type Column  in common/types/ui/column.type.ts
```
type Column<Data extends Record<string, unknown>> = {
  header: string
  accessor: keyof Data;
}
```
```suggestion

type Props<Data extends Record<string, unknown>> = {
  columns: Column<Data>[];
  data: Data[];
```